addons/sale/tests/test_sale_order_discount.py

```diff
@@ -141,3 +141,43 @@ def test_line_and_global_discount(self):
         })
         self.wizard.action_apply_discount()
         self.assertEqual(self.sale_order.amount_untaxed, amount_with_line_discount * 0.9)
+
+    def test_apply_mixed_epd_discount(self):
```

Comment:
`test_sale_order_discount` is a test file dedicated to the `sale.order.discount` wizard.

Please move your test to another test file (maybe `test_taxes_tax_totals_summary` ?)